apps/data/lib/data/repo.ex

```diff
@@ -1,5 +1,6 @@
 defmodule Data.Repo do
   use Ecto.Repo, otp_app: :data
+  use Scrivener, page_size: 25
```

Comment:
I just realized that this is off - going to remove it, this is leftover from when first playing with the library.